packages/components/src/components/domain/unifiedSearch/graphql/unifiedSearchMap.query.ts

```diff
@@ -17,14 +17,14 @@ export const SEARCH_MAP_QUERY = gql`
     $orderByName: OrderByName
   ) {
     unifiedSearch(
-      q: $q
-      index: "location"
```

Comment:
what is this change? What does it do?

Reply:
what I meant was that the index earlier got "", but now does not

Reply:
The `index`'s type changed from `String` to `enum UnifiedSearchIndex`:
```graphql
  enum UnifiedSearchIndex {
    administrative_division
    helsinki_common_administrative_division
    ontology_tree
    ontology_word
    event
    location
  }
```
That's what necessitated the removal of the quotes around "location" here.